README.md

```diff
@@ -213,7 +213,45 @@ The current version of Lumberjack requires:
 - If you **have a feature request**, open an issue.
 - If you **want to contribute**, submit a pull request.
 
+### Data Collection Practices
+
+Per  [App privacy details on the App Store](https://developer.apple.com/app-store/app-privacy-details/), Apple is requesting app developers to provide info about their data collection, us SDK maintainers must provide them with the same data.
+
+#### Data collection by the framework
+
+**By default, CocoaLumberjack does not collect any data on its own.** 
```

Comment:
Maybe it is better to emphasize this via quote? If you open "Display rich diff" view, you will find that two bold lines one below another will interfere and will hide second line.

Reply:
Something like this

#### Data collection by the framework

```
 By default, CocoaLumberjack does NOT collect any data on its own.
```

Reply:
Still not the best option, but, it separates header and body.

Reply:
I have converted all our headings from h3 to h2 (which was available), thus making "Data collection by the framework" an h3 which is much more distinct than an h4. Let me know if this is ok now